cpp/src/traversal/tsp.hpp

```diff
@@ -25,9 +25,17 @@
 
 namespace cugraph {
 namespace detail {
+
+struct TSPResults {
+  float **best_x_pos;
```

Comment:
Why not using rmm uvector?

Reply:
Since I'm updating the struct inside a kernel I prefer storing raw pointers here. The other option I see is to have device_vectors in the struct and pass the pointers as parameters

Reply:
that's ok